fearless/Modules/Staking/ValidatorInfo/ValidatorInfoViewModel/ValidatorInfoViewModelFactory.swift

```diff
@@ -0,0 +1,163 @@
+import Foundation
+import FearlessUtils
+import SoraFoundation
+import CommonWallet
+
+protocol ValidatorInfoViewModelFactoryProtocol {
+    func generateExtrasViewModel(
```

Comment:
I think we already use "create" verb in the factories. I suggest to rename

Reply:
Sure thing

Reply:
Done